scripts/deploy/execute-deployment-pipeline.mjs

```diff
@@ -26,46 +44,70 @@ function getVersionComposants(version) {
   ];
 }
 
-function getResolveVariableString(version, packageName) {
+function getVersionsSubpaths(version) {
   const prNumber = process.env.PR_NUMBER;
   const versionComposantsOrdered = getVersionComposants(version);
```

Comment:
Related to https://github.com/coveo/ui-kit/pull/5837/files#r2256929752, I'd rename this variable to `versionComponentsOrdered`